scripts/src/models/users.ts

```diff
@@ -1,7 +1,7 @@
-import { Column, Entity } from "typeorm";
+import { BaseEntity, Column, Entity, Index } from "typeorm";
```

Comment:
Why did you add these imports?
Looks like you're not using any of those:
`BaseEntity`, `Index`, `Model`, `generateId`

Reply:
I know that IntelliJ removes unused imports in Java. Can it do it for javascript too?

Reply:
`ctrl-alt-o` optimize imports. but you can't automate it in pycharm :(